services/provider/client/client.go

```diff
@@ -55,14 +55,15 @@ func (cc *OCSProviderClient) Close() {
 
 // OnboardConsumer to validate the consumer and create StorageConsumer
 // resource on the StorageProvider cluster
-func (cc *OCSProviderClient) OnboardConsumer(ctx context.Context, ticket, name string) (*pb.OnboardConsumerResponse, error) {
+func (cc *OCSProviderClient) OnboardConsumer(ctx context.Context, ticket, name, operatorVersion string) (*pb.OnboardConsumerResponse, error) {
```

Comment:
This approach is not flexible, adding more variables to signature is limiting as the number of onboard params will grow. Can we move to a similar approach we utilized for the report status mechanizem?

Reply:
done.